lib/kdump_utils.pm

```diff
@@ -434,7 +434,7 @@ sub check_function {
         # check the core dump via the crash utility if possible
         my $crash_cmd;
         my $vmcore_glob = '/var/crash/*/vmcore';
-        my $vmlinux_glob = (is_sle("<16") || is_sle_micro || is_leap("<16.0"))
+        my $vmlinux_glob = (is_sle("<16") || is_sle_micro("<6.0") || is_leap("<16.0"))
           ? '/boot/vmlinux-$(uname -r)*'
           : '/usr/lib/modules/$(uname -r)/vmlinux*';
```

Comment:
How about this? It would allow not using any `is_*` function:
```
my $vmlinux_glob = 'ls /boot/vmlinux-$(uname -r)* /usr/lib/modules/$(uname -r)/vmlinux* 2>/dev/null'
```
I see later `head -n1` is used, otherwise I would add `tail -1` to limit number of results.

Reply:
Are you sure it will work  and will not complain about missing file? I would rather do it in other refactoring.

Reply:
Will follow in  https://progress.opensuse.org/issues/151774

Reply:
personally, while this could be considered superfluous -> (is_sle("<16") || is_sle_micro("<6.0") || is_leap("<16.0")), I find it useful for product testing -> once quickly and explicitly read what is expected in which product and its version; ofc this is only my opinion and my 2c

Reply:
We can follow up discussion in above ticket(when we will get to it).

Reply:
@schlad sure, np.